client/landing/gutenboarding/onboarding-block/acquire-intent/site-title.tsx

```diff
@@ -52,7 +55,10 @@ const SiteTitle: React.FunctionComponent< Props > = ( { onSubmit } ) => {
 					autoCorrect="off"
 					data-hj-whitelist
 				/>
-				{ /* @TODO: add info tip here */ }
+				<p className={ classnames( 'site-title__input-hint', { 'has-site-title': !! siteTitle } ) }>
+					<Icon icon={ tip } size={ 14 } />
+					<span>{ __( "Don't worry, you can change it later." ) }</span>
```

Comment:
If we don't change the text (https://github.com/Automattic/wp-calypso/pull/43317#issuecomment-644638361), this text could explained with "translation" comment so that it's clearer what "it" refers to. It might change how it gets translated in some languages.

Reply:
Good tip!